static/app/components/idBadge/getBadge.tsx

```diff
@@ -4,6 +4,7 @@ import BaseBadge from 'sentry/components/idBadge/baseBadge';
 import MemberBadge from 'sentry/components/idBadge/memberBadge';
 import OrganizationBadge from 'sentry/components/idBadge/organizationBadge';
 import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+// This imports a base badge, should we instead import teamBadge/index?
```

Comment:
Is this intended, I left a comment to not forget

Reply:
That might have been a mistake, the TeamBadge wrapper includes reading from the store as the team avatar changes.

Reply:
Perfect, I will replace it with the correct import and remove my comment, thanks!